source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -4,33 +4,18 @@ enum ActionStyle: String {
     case positive
     case `default`
     case destructive
-    case none
+    case darkMode
     case inline
 }
 
 class ACRButton: FlatButton, ImageHoldingView {
-    struct Constants {
-        static let blueColorCode = "#007EA8"
-        static let darkBlueColorCode = "#0A5E7D"
-        
-        static let greenColorCode = "#1B8728"
-        static let darkGreenColorCode = "#196323"
-        
-        static let redColorCode = "#D93829"
-        static let darkRedColorCode = "#A12C23"
-        
-        static let grayColorCode = "#E6E8E8"
-        static let darkGrayColorCode = "#B4B6B8"
-    }
-    
-    public var backgroundColor: NSColor = .init(red: 0.35216, green: 0.823529412, blue: 1, alpha: 1)
-    public var hoverBackgroundColor: NSColor = .linkColor
-    public var activeBackgroundColor: NSColor = .linkColor
+    public var backgroundColor: NSColor = .clear
     
     private var inActiveTextColor: NSColor = .linkColor
     private var inActiveborderColor: NSColor = .linkColor
     private var hoverButtonColor: NSColor = .linkColor
-    private var buttonStyle: ActionStyle = .default
+    private var buttonActionStyle: ActionStyle = .default
+    private var buttonConfig: ButtonConfig = .default
     
```

Comment:
Guess we need not store this. 
These can be just method params to `setupButtonStyle` method.

Reply:
okay, will remove buttonConfig but buttonActionStyle is being used actually to set the roundedBorders, because in the layout we are adding roundedBorder logic